spec/pundit_spec.rb

```diff
@@ -33,6 +33,26 @@
       expect(Pundit.authorize(user, [:project, comment], :update?)).to be(comment)
     end
 
+    it "returns the record when passed record with nested namespace " do
+      expect(Pundit.authorize(user, [:project, :admin, comment], :update?)).to be(comment)
+    end
+
+    it "can use headless policy" do
+      expect(Pundit.authorize(user, :publication, :create?)).to be_truthy
+    end
+
+    it "returns the policy name symbol when passed record with headless policy" do
+      expect(Pundit.authorize(user, :publication, :create?)).to be(:publication)
+    end
+
+    it "can use without a particular instance" do
+      expect(Pundit.authorize(user, Post, :show?)).to be_truthy
+    end
+
+    it "returns the class when passed record not a particular instance" do
+      expect(Pundit.authorize(user, Post, :show?)).to be(Post)
```

Comment:
`be` feels unusual to me - would `eq?` be more expressive/conventional? 
The same for "comment" above

Reply:
OK!